src/main/java/io/unlogged/command/ServerMetadata.java

```diff
@@ -9,11 +13,24 @@ public void setAgentServerPort(int agentServerPort) {
     String includePackageName;
     String agentVersion;
     String agentServerUrl;
+	private String hostname;
+	private long createdAt;
+
     public ServerMetadata(String includePackageName, String agentVersion, int agentServerPort) {
         this.includePackageName = includePackageName;
         this.agentVersion = agentVersion;
+		this.createdAt = System.currentTimeMillis() / 1000L;
```

Comment:
why / 1000 ? can record the timeMillis as it is

Reply:
This is now set as ms.